src/autopas/tuning/tuningStrategy/ruleBasedTuning/RuleBasedTuning.h

```diff
@@ -145,10 +158,14 @@ class RuleBasedTuning : public TuningStrategyInterface {
    */
   std::vector<rule_syntax::ConfigurationOrder> applyRules(const std::vector<Configuration> &searchSpace);
 
+  std::vector<rule_syntax::ConfigurationOrder> _lastApplicableConfigurationOrders;
```

Comment:
```suggestion
  std::vector<rule_syntax::ConfigurationOrder> _lastApplicableConfigurationOrders{};
```
missing initialization
- [ ] same for `_traversalTimes` if I see this correctly